Source/Validation.swift

```diff
@@ -25,38 +25,33 @@
 import Foundation
 
 extension Request {
-
-    /**
-        Used to represent whether validation was successful or encountered an error resulting in a failure.
-
-        - Success: The validation was successful.
-        - Failure: The validation failed encountering the provided error.
-    */
+    /// Used to represent whether validation was successful or encountered an error resulting in a failure.
+    ///
+    /// - success: The validation was successful.
+    /// - failure: The validation failed encountering the provided error.
     public enum ValidationResult {
         case success
         case failure(NSError)
     }
 
-    /**
-        A closure used to validate a request that takes a URL request and URL response, and returns whether the
-        request was valid.
-    */
-    public typealias Validation = (Foundation.URLRequest?, HTTPURLResponse) -> ValidationResult
-
-    /**
-        Validates the request, using the specified closure.
-
-        If validation fails, subsequent calls to response handlers will have an associated error.
-
-        - parameter validation: A closure to validate the request.
-
-        - returns: The request.
-    */
+    /// A closure used to validate a request that takes a URL request and URL response, and returns whether the
+    /// request was valid.
+    public typealias Validation = (URLRequest?, HTTPURLResponse) -> ValidationResult
+
+    /// Validates the request, using the specified closure.
+    ///
+    /// If validation fails, subsequent calls to response handlers will have an associated error.
+    ///
+    /// - parameter validation: A closure to validate the request.
+    ///
+    /// - returns: The request.
     @discardableResult
-    public func validate(_ validation: Validation) -> Self {
+    public func validate(validation: Validation) -> Self {
```

Comment:
Fine by me. It will almost always be used in a trailing closure which is why I removed it. Addressed in 8c5a83bc.